### PR TITLE
frozen<...> support

### PR DESCRIPTION
**Pull request by rashtao**
This should allow queries like this:

```js
var cql = cassanKnex("ks")
    .createColumnFamilyIfNotExists("cf")
    .text("textType")
    .frozen("frozen_field", "list<text>")
    .primary("textType");
```

### CODE REVIEW

**Comment by UnbounDev** [comment]
@rashtao thanks for the contribution! Can you provide an appropriate test case w/i the [keyspace test file](https://github.com/azuqua/cassanknex/blob/master/tests/keyspace.test.js)? Tests can be run using `grunt`; additionally, you can test the execution of the query against a live cassandra instance by adding a use case to the [live test file](https://github.com/azuqua/cassanknex/blob/master/tests/live/live.test.js) and run using `grunt test-live` (if you'd like, I'm satisfied w/ seeing an appropriately formatted CQL statement generated and checked by the keyspace tests).

File: componentDeclarations/componentBuilderMethods.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ module.exports = {
   "columnFamily": {
     "primary": {"name": "primary", "grouping": "column", jsType: "PRIMARY_KEY"},
 
+    "frozen": {"name": "frozen", "grouping": "column", jsType: "frozen"},
```
Comment (UnbounDev) [inline componentDeclarations/componentBuilderMethods.js]: `jsType: "frozen"` should be `jsType: "object"` - see http://docs.datastax.com/en/developer/nodejs-driver/2.1/nodejs-driver/reference/userDefinedTypes.html

**Comment by rashtao** [comment]
I wrote the tests and implemented also a basic support for UDTs creation. I have just duplicated the ColumnFamily code, so since types DDL is just a subset of tables DDL (right?!?!) we already have all the methods implemented. Of course there is a subset of methods that are valid for tables but not for types... So maybe we could work a bit more on the refinements of this...

**Comment by UnbounDev** [comment]
@rashtao yes - we should discuss and refine before pushing a release. Can you close this PR and re-open it against the [develop](https://github.com/azuqua/cassanknex/tree/develop) branch (I'll merge it there)? I'd like to clone and work on the updates/test; when it's ready we'll merge w/ master (gitflow ftw).

**Pull request closed without merging**